### DISCUSSION TITLE
Sentiment Analytics Reference

### DISCUSSION THREAD

**Discussion by naturallyfunny** [teamalgoritma/community#790] [Q&A] [answered]
Ada yang tahu tutorial untuk metode klasifikasi sentiment seperti di [video DCD ini](https://www.youtube.com/watch?v=ezQMIldHT-k)?

**Comment by saskia-dwi-ulfah** [comment, marked as answer]
Dear @ArdiantheGreat 

Pada video tersebut, teknik klasifikasi sentiment yang digunakan adalah NRC Lexicon. 
Berikut beberapa referensi yang bisa dieksplor:

1. [Sentiment analysis with tidy data](https://www.tidytextmining.com/sentiment#:~:text=The%20nrc%20lexicon%20categorizes%20words,into%20positive%20and%20negative%20categories.)
2. [NRC word-emotion association lexicon](https://search.r-project.org/CRAN/refmans/textdata/html/lexicon_nrc.html)
3. [Chapter 4 Sentiment Analysis](https://bookdown.org/psonkin18/berkshire/sentiment.html)
4. [Sentiment Analysis with 'syuzhet' using R](https://programminghistorian.org/en/lessons/sentiment-analysis-syuzhet)

Semoga beberapa referensi ini dapat membantu.

Best regards,
Team algoritma
